_component_design/tabset.md

```diff
@@ -17,34 +17,95 @@ helix-ui-javascript: true
 
 <section class="static-section" markdown="1">
 
-## Introduction
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use tabs in the following situations:
+
+- Users need to alternate between different views, while the context remains constant.
+- You want the content of a page to be scannable and easy to navigate. Tabs help users quickly find what they need.
+- You do not need to create a single, unified experience for the user. Tabs eliminate the need for the user to scroll through long pages to find the right section.
+
+Do not use tabs for the following situations:
+
+- Tabs are not suited for use for navigation
```

Comment:
Consider changing to something like: 

`Tabs are not suited for use as primary navigation for an entire page.`